src/firefly/java/edu/caltech/ipac/firefly/core/background/ScriptAttributes.java

```diff
@@ -12,5 +12,5 @@
 /**
 * @author Trey Roby
 */
-public enum ScriptAttributes {URLsOnly, Unzip, Curl, Wget}
+public enum ScriptAttributes {URLList, Unzip, Curl, Wget}
```

Comment:
What was the reason for changing this enum? I believe it’s also referenced in several JS/JSX files, but I don’t see those updates here. Are you sure you tested all of them?

Reply:
But I'm open to reverting this change (keeping the enum the same) for only passing in a different one for URLs when calling `makeScriptFilename`

Reply:
Generally, you shouldn’t rename an enumeration just to change how it’s labeled. However, if the intent is to refactor because the existing name is misleading, then it makes sense to rename it globally.
I noticed references on the client side, in BackgroundUtil.js and ScriptDownloadDialog.jsx, and there may be more. Please test after making the modifications.